Content/.template.config/ide.host.json

```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "http://json.schemastore.org/vs-2017.3.host",
+    "symbolInfo": [
+        {
+            "id": "githubUsername",
+            "name": {
+                "id": "githubUsername",
+                "text": "Github Username"
```

Comment:
this is how the short names get replaced to longer user-friendly names in an IDE like VS.